### DISCUSSION TITLE
Any VPS Deploy Tutorial Available ?

### DISCUSSION THREAD

**Discussion by JOHN748** [anasty17/mirror-leech-telegram-bot#1213] [Q&A]
I came to know heroku branch is removed because of some issues, so I don't have any other option than deploying in vps. Can anybody share detailed deployment tutorial for VPS? 

And if there is any tutorial already available then please share it! ty

**Comment by anasty17** [comment]
Why need tutorial?
Everything in readme.
If u have problem in any step then ask

**Comment by JOHN748** [comment]
Why Not?
Not Everything in readme is not in detailed as you can see!
Everything that you've mentioned in readme about VPS deployment are may be understandable to the people who have some experience in it or some newbie. But I've never heard or into these things before. So all I just ask is some detailed tutorial in article/video form to understand and apply. And I can understand you cannot able to manage some noobs like me and out there! So, I leave it to people who see this discussion who are willing to help can make the above I've requested. Ty :)

**Reply by anasty17** [reply]
Bruh it's just small difference btw deploy on vps and heroku.
Install docker from official site for linux arch(ur vps arch)
Clone the repo.
Fill required vars.
Now follow to deploy on vps in readme. Its simple
If u want to use docker compose then steps.

**Comment by JOHN748** [comment]
Sucessfully deployed my bot bro. Everything working well and your bot is osm. ;)
One last qn. When I deployed my bot I frgt to extend the size of the strg initally it was at 8GB when I deployed. After that I realized that my strg is getting out of disk, So I've extended my disk strg to 29GB but in my bot it's still showing the previous disk space (8GB). Can you please help me solve this ? [I'm Using AWS]
![image](https://user-images.githubusercontent.com/48284893/202292756-606abbd7-9b80-466f-a01a-c894c0582ae3.png)
![Untitled](https://user-images.githubusercontent.com/48284893/202293750-94dd261f-ae3c-4b34-863b-27a43612ed40.png)

**Reply by anasty17** [reply]
https://stackoverflow.com/questions/24309526/how-to-change-the-docker-image-installation-directory

**Comment by JOHN748** [comment]
LEECH_SPLIT_SIZE is not changing bro, I'm having premium acct so I've entered 4GB in bytes in edit variable but it's not changing.
![image](https://user-images.githubusercontent.com/48284893/202329782-d64dcebd-1abb-4f4b-9566-f43e0c1b27a8.png)

**Comment by anasty17** [comment]
Because u didn't create new user session string after premium subscription. And after u change the user session u must restart the bot

**Reply by JOHN748** [reply]
Generated new and now it's working bro. Can you pls help me on the above qn ? About disk space!
